pyproject.toml

```diff
@@ -1,13 +1,12 @@
 [build-system]
 # Minimum requirements for the build system to execute.
 requires = [
-    "setuptools<60.0",
+    "setuptools>=40.8.0",
```

Comment:
Use please `setuptools>=46.4,<60.0` - the upper bound is required for numpy.distutils to work.

Reply:
Changed in next commit.